pubspec.yaml

```diff
@@ -73,10 +73,10 @@ flutter:
   # "family" key with the font family name, and a "fonts" key with a
   # list giving the asset and other descriptors for the font. For
   # example:
-  # fonts:
-  #   - family: Schyler
-  #     fonts:
-  #       - asset: fonts/Schyler-Regular.ttf
+  fonts:
```

Comment:
Try using google_fonts flutter package, which is said to download the fonts online and store in cache and use it for rest of times.

This is done to minimise the app size in play store